doordeck-sdk/src/commonMain/kotlin/com/doordeck/multiplatform/sdk/internal/api/AccountlessClient.kt

```diff
@@ -30,14 +35,17 @@ internal open class AccountlessClient(
      * @see <a href="https://developer.doordeck.com/docs/#registration-v3">API Doc</a>
      */
     suspend fun registrationRequest(email: String, password: String, displayName: String?, force: Boolean): TokenResponse {
-        return httpClient.post(Paths.getRegistrationPath()) {
+        return httpClient.post<TokenResponse>(Paths.getRegistrationPath()) {
             addRequestHeaders(apiVersion = ApiVersion.VERSION_3)
             setBody(RegisterRequest(
                 email = email,
                 password = password,
                 displayName = displayName
             ))
             parameter(Params.FORCE, force)
+        }.also {
+            contextManager.setAuthToken(it.authToken)
+            contextManager.setRefreshToken(it.refreshToken)
```

Comment:
From now on, we will update the context values whenever possible. Previously, this had to be done manually by the developer. Please note that this only updates the variable; it does not persist it.